semantic error: 'i32[3]' object does not support item assignment
 --> tests/errors/test_pointer_types.py:6:5
  |
6 |     y[0] = 1
  |     ^^^^^^^^ 
